README.md

```diff
@@ -71,3 +71,22 @@ Outside a container: Run
 ```bash
 npm run start
 ```
+
+## Deployment
+
+### To `dev`
+`stela` deploys to `dev` automatically upon any merge to `main`.
+
+### Beyond `dev`
+To deploy to `staging` and `prod`, create a Release with a tag of the form `vX.X.X` (this should conform to semantic
+versioning). This will trigger a deploy to staging. If that is successful, the workflow will pause and wait for manual
+approval to deploy to prod. Given manual approval, it will deploy to prod.
```

Comment:
So this should be updated to say that releases don't automatically trigger a deploy to staging? Or am I misunderstanding your comment above? I guess I've *seen* releases automatically trigger a deploy to staging, so I know that works.

Reply:
Releases still trigger a deploy to staging, there just isn't a way to do deploys that are _only_ going to staging through releases